doc/user-manual/language/safe-agda.lagda.rst

```diff
@@ -51,22 +51,25 @@ then all its imported modules must also be declared safe.
 
 .. NOTE::
 
-   The ``--safe`` option turns off the ``--guardedness`` and
-   ``--sized-types`` options that are both on by default. One of them
-   (but not both, see above) can be turned on again by giving either
-   ``--guardedness`` or ``--sized-types`` *after* the  ``--safe`` option, e.g.
+   The ``--guardedness`` and ``--sized-types`` options are both on by default.
+   Setting the ``--safe`` option will ignore these defaults and only work if at most
```

Comment:
"and only work" could perhaps give the wrong impression, how about something like:
```
The ``--guardedness`` and ``--sized-types`` options are both on by default. 
However setting the ``--safe`` option will make Agda turn them off, unless they
are explicitly turned on. Turning on more than one of ``--guardedness`` and
``--sized-types`` together with ``-safe`` results in an error, since these options
are currently inconsistent.